### PR TITLE
Feature/eema1 nrl 767 skip permissions check for sync

### PR DESCRIPTION
**Pull request by eesa456**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js [hunk 1]
```diff
@@ -30,39 +30,56 @@
...
+      hasAllPointerPermissions = true;
+    }
+    nrlPermissions = permissions;
+  }
+
+  var enableAuthorizationLookup = context.getVariable("app.enable-authorization-lookup");
   if(enableAuthorizationLookup == "true") {
     enableAuthorizationLookup = true
-  } else if (enableAuthorizationLookup === null) {
+  } else if (enableAuthorizationLookup === null  || enableAuthorizationLookup == "false") {
```
Comment (axelkrastek1-nhs) [inline proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js:55]: This fix needs to be applied to the consumer proxy as well.

Comment (eesa456) [reply proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js:55]: yes

Comment (eesa456) [reply proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js:55]: https://github.com/NHSDigital/nrl-consumer-api/pull/126

Comment (eesa456) [reply proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js:55]: if you would kindly review

**Review by axelkrastek1-nhs — APPROVED** [review]

**Pull request merged**